.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,7 +33,7 @@ is also intended for other testing-related comments. -->
 that might be useful for reviewers.. -->
 
 ## Acknowledgements
-- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
+- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/stable/contributing.html)
```

Comment:
I also wouldn't change it here since this is for the contributors who definitely should read our latest version of the contributing.rst